src/WebJobs.Script/Workers/Rpc/RpcInitializationService.cs

```diff
@@ -59,7 +59,7 @@ public async Task StartAsync(CancellationToken cancellationToken)
 
         public async Task StopAsync(CancellationToken cancellationToken)
         {
-            _logger.LogDebug("Shuttingdown Rpc Channels Manager");
+            _logger.LogDebug("Shutting down Rpc Channels Manager");
```

Comment:
Curious, do we have any process around keeping our detectors updated when changing log messages (if any detector is dependent on this log line)?
cc @kashimiz

Reply:
Good question, I am also worried about that. Happy to revert for now and have a cleanup PR in future